meta-resin-common/recipes-support/resin-mounts/resin-mounts-flasher/etc-dnsmasq.mount

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Resin dnmasq configuration
+Requires=mnt-conf.mount
+After=mnt-conf.mount
+Before=dnsmasq.service connman.service
```

Comment:
Well connman needs to write to this location the nameserves that it discovers, so this location has to be ready for connman to write.